src/main/scala/centaur/test/metadata/WorkflowMetadata.scala

```diff
@@ -52,7 +53,7 @@ object WorkflowMetadata {
   def fromConfig(config: Config): ErrorOr[WorkflowMetadata] = {
     config.extract[Map[String, String]] match {
       case Result.Success(m) => Valid(WorkflowMetadata(m mapValues { JsString(_) }))
-      case Result.Failure(_) => invalidNel("Metadata block can not be converted to a Map")
+      case Result.Failure(_) => invalidNel(s"Metadata block can not be converted to a Map: $config")
```

Comment:
It's way too spammy though. 

The problem is there's not much useful info in this section w/o bringing something in for the sole purpose of printing out an error message somewhere